harfbuzz-sys build fails with cannot find input file: `test/Makefile.in'

I just tried to clone the repo and run `cargo build`, and I got this build error when compiling the harfbuzz-sys crate dependency:

```
--- stderr
config.status: error: cannot find input file: `test/Makefile.in'
make: *** [all] Error 1
thread 'main' panicked at 'assertion failed: Command::new("make").env("MAKEFLAGS",
                         env::var("CARGO_MAKEFLAGS").unwrap_or_default()).args(&["-R",
                                                                                 "-f",
                                                                                 "makefile.cargo"]).status().unwrap().success()', /usr/local/google/home/clupo/.cargo/registry/src/github.com-1ecc6299db9ec823/harfbuzz-sys-0.3.1/build.rs:31:9
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed
```

This happens when I compile skribo on Debian Linux. However, when I do it on my Mac it builds without the error above.

These may be related:
https://github.com/servo/rust-harfbuzz/issues/137
https://github.com/servo/rust-harfbuzz/pull/136

Note that if I build rust-harfbuzz directly from source (as opposed to fetching it as a dependency of skribo), it builds without that error, also on Linux.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -11,9 +11,8 @@ readme = "README.md"
 categories = ["graphics", "text-processing"]
 
 [dependencies]
-# Note: this should be updated to 0.3.2 when it's released.
-harfbuzz = "=0.3.0"
-harfbuzz-sys = "=0.3.0"
+harfbuzz = "0.3.1"
+harfbuzz-sys = "0.3.2"
 euclid = "0.19"
 font-kit = { version = "0.1.0"} #, features = ["loader-freetype-default"] }
 unicode-normalization = "0.1.8"

--- a/src/harfbuzz.rs
+++ b/src/harfbuzz.rs
@@ -1,22 +1,17 @@
 //! A HarfBuzz shaping back-end.
 
-use std::os::raw::{c_char, c_uint, c_void};
-use std::sync::Arc;
-
 use euclid::Vector2D;
 
 use harfbuzz::sys::{
-    hb_blob_create, hb_blob_destroy, hb_blob_t, hb_buffer_get_glyph_infos,
+    hb_buffer_get_glyph_infos,
     hb_buffer_get_glyph_positions, hb_face_create, hb_face_destroy, hb_face_reference, hb_face_t,
     hb_font_create, hb_font_destroy, hb_position_t, hb_shape,
 };
-use harfbuzz::sys::{HB_MEMORY_MODE_READONLY, HB_SCRIPT_DEVANAGARI};
-use harfbuzz::{Buffer, Direction, Language};
-
-use font_kit::loaders::default::Font;
+use harfbuzz::sys::{HB_SCRIPT_DEVANAGARI};
+use harfbuzz::{Blob, Buffer, Direction, Language};
 
 use crate::unicode_funcs::install_unicode_funcs;
-use crate::{FontCollection, FontRef};
+use crate::{FontRef};
 use crate::{Glyph, Layout, TextStyle};
 
 struct HbFace {
@@ -26,7 +21,7 @@ struct HbFace {
 impl HbFace {
     pub fn new(font: &FontRef) -> HbFace {
         let data = font.font.copy_font_data().expect("font data unavailable");
-        let blob = ArcVecBlob::new(data);
+        let blob = Blob::new_from_arc_vec(data);
         unsafe {
             let hb_face = hb_face_create(blob.into_raw(), 0);
             HbFace { hb_face }
@@ -128,48 +123,3 @@ unsafe extern "C" fn font_table_func(
     unimplemented!()
 }
 */
-
-/// A HarfBuzz blob that's backed by an `Arc<Vec>`.
-///
-/// Note: this can probably be merged with `Blob` in the harfbuzz crate.
-struct ArcVecBlob(*mut hb_blob_t);
-
-impl ArcVecBlob {
-    pub fn new(data: Arc<Vec<u8>>) -> ArcVecBlob {
-        let len = data.len();
-        assert!(len <= c_uint::max_value() as usize);
-        unsafe {
-            let data_ptr = data.as_ptr();
-            let ptr = Arc::into_raw(data);
-            let hb_blob = hb_blob_create(
-                data_ptr as *const c_char,
-                len as c_uint,
-                HB_MEMORY_MODE_READONLY,
-                ptr as *mut c_void,
-                Some(arc_vec_blob_destroy),
-            );
-            ArcVecBlob(hb_blob)
-        }
-    }
-
-    pub fn into_raw(self) -> *mut hb_blob_t {
-        let ptr = self.0;
-        std::mem::forget(self);
-        ptr
-    }
-}
-
-// Can implement Clone, Deref as needed; impls similar to harfbuzz crate
-
-impl Drop for ArcVecBlob {
-    fn drop(&mut self) {
-        unsafe {
-            hb_blob_destroy(self.0);
-        }
-    }
-}
-
-// This has type hb_destroy_func_t
-unsafe extern "C" fn arc_vec_blob_destroy(user_data: *mut c_void) {
-    std::mem::drop(Arc::from_raw(user_data as *const Vec<u8>))
-}

